pipelines/setup.yml

```diff
@@ -141,11 +157,61 @@ jobs:
           put: release-ci-tfstate
           params:
             file: updated-release-ci-tfstate/release-ci.tfstate
+
+  - name: generate-ssh-keys
+    serial: true
+    plan:
+      - aggregate:
+         - get: ssh-private-key
+         - get: ssh-public-key
+         - get: pipeline-trigger
+           passed: ['terraform']
+           trigger: true
+      - task: generate-deployments-ssh-keypair
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/git-ssh
+              tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
+          inputs:
+            - name: ssh-private-key
+            - name: ssh-public-key
+          outputs:
+            - name: generated-ssh-private-key
+            - name: generated-ssh-public-key
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                if [ -s ssh-private-key/ci_build_tag_key ] ; then
+                  echo "Deployments private key non-zero size, skipping generation..."
+                  echo "Key uploads will fail and this is OK as no new keys have been generated."
+                  exit 0
```

Comment:
Can this job output the public key so that you can copy/paste it without manually looking in the bucket?

It would be good to do this when the key is first generated below too.